pkgs/development/compilers/idris2/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchFromGitHub, makeWrapper
+, clang, chez 
+}:
+
+# Uses scheme to bootstrap the build of idris2
+stdenv.mkDerivation {
+  name = "idris2";
+  version = "0.2.0-840e020";
+
+  nativeBuildInputs = [ makeWrapper ];
+  buildInputs = [ clang chez ];
```

Comment:
If chez is used at build time it needs to be in `nativeBuildInputs`. Same thing for `clang`. I am not sure if you also need their runtime libraries. You can check by set `strictDeps`.

Reply:
Ah, good point. Thanks!